xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd

```diff
@@ -150,10 +150,13 @@ Rail transport, Roads and Road transport
 		<xsd:restriction base="xsd:NMTOKEN">
 			<xsd:enumeration value="unknown"/>
 			<xsd:enumeration value="lift"/>
+			<xsd:enumeration value="wheelchairLift"/>
```

Comment:
The changes in netex_facility_support were already made. Only netex_facility_version is missing. I think we can drop the changes to netex_facility_support.xsd

Reply:
@skinkie

Reply:
@skinkie

Reply:
sorry it was Christophe @Aurige

Reply:
sorry it was Christophe @Aurige